src/Umbraco.Web.UI.Client/src/views/components/umb-mini-search/umb-mini-search.html

```diff
@@ -1,9 +1,10 @@
 <ng-form class="umb-mini-search" ng-class="{'--has-value': vm.model !== null && vm.model !== ''}" novalidate>
-    <i class="icon icon-search" aria-hidden="true"></i>
+    <label for="mini-search"><i class="icon icon-search"></i><span class="sr-only"><localize key="general_typeToSearch">Search...</localize></span></label>
     <input
         class="form-control search-input"
         type="text"
         localize="placeholder,label"
+        id="mini-search"
```

Comment:
I wonder what happens if we have multiple mini searches on one page.. not sure it can happen? Can we make this unique for each instance somehow you think?

Reply:
I wondered the same. I know currently I don't have any situation where two show up at the same time. I wasn't cure where to get an id to make it unique, but as I am working on another issue I have learned a little more about the structure of AngularJS and I think I can pull out something to make it unique. I will circle back to this after I finish that issue.